File: objectorientedprogramming/student.py
class Student:
    def setStudent(self,rol,name,course,total):
        self.rol=rol
        self.name=name
        self.course=course
        self.total=total
    def printStudent(self):
        print(self.rol)
        print(self.name)
        print(self.course)
        print(self.total)
obj=Student()
obj.setStudent(4,"samvi","eng",500)
obj.printStudent()

#self is a keyword to point current obj instance variables

File: a.objectoriented.js/student.js
class Student{
  setStudent(name,rollno,course,total){
  this.name=name;
  this.rollno=rollno;
  this.course=course;
  this.total=total;
  }
  printStudent(){
  console.log(this.name)
  console.log(this.rollno)
  console.log(this.course)
  console.log(this.total)
  }
}
var obj=new Student()
obj.setStudent("samvi",25,"eng",500)
obj.printStudent()